lua/wire/client/hlzasm/hc_codetree.lua

```diff
@@ -90,7 +90,11 @@ end
 -- Returns free (non-busy) register. Does not check EBP and ESP
 function HCOMP:FreeRegister()
   -- Try to find a free register
-  for i=1,6 do
+  for i=1,#self.RegisterBusy do
+    if not self.RegisterBusy[i] then return i end
+  end
+
+    for i=1,6 do
```

Comment:
Can this loop be deleted?

Reply:
Keen eye, thank you, this probably showed up cause I deleted my code and had to recover it by ctrl+z'ing in my vscode and correcting all of the errors that popped up, does the fix I've just pushed look good to you?